framework/wazuh/core/mitre.py

```diff
@@ -79,26 +88,41 @@ def _format_data_into_dictionary(self):
 
 
 class WazuhDBQueryMitreMetadata(WazuhDBQueryMitre):
+    TABLE_NAME = 'metadata'
 
     def __init__(self):
         """Create an instance of WazuhDBQueryMitreMetadata query."""
 
         min_select_fields = {'key', 'value'}
         fields = {'key': 'key', 'value': 'value'}
 
-        WazuhDBQueryMitre.__init__(self, table='metadata', min_select_fields=min_select_fields, fields=fields,
-                                   default_sort_field='key')
+        WazuhDBQueryMitre.__init__(self, table=self.TABLE_NAME, min_select_fields=min_select_fields, fields=fields,
+                                   default_sort_field=MAIN_TABLES_PKS[self.TABLE_NAME])
 
     def _filter_status(self, status_filter):
         pass
 
 
 class WazuhDBQueryMitreRelational(WazuhDBQueryMitre):
+    TABLE_FIELDS = {
+        'phase': {'min_select_fields': {'tactic_id', 'tech_id'},
+                  'default_sort_field': 'tactic_id',
+                  'fields': {'tactic_id': 'tactic_id', 'tech_id': 'tech_id'}},
+        # source_id refers to mitigation_id, group_id or software_id
+        # target_id always refers to technique_id
+        'mitigate': {'min_select_fields': {'source_id', 'target_id'},
+                     'default_sort_field': 'source_id',
+                     'fields': {'source_id': 'source_id', 'target_id': 'target_id'}},
+        'use': {'min_select_fields': {'source_id', 'target_id'},
+                'default_sort_field': 'source_id',
+                'fields': {'source_id': 'source_id', 'source_type': 'source_type',
+                           'target_id': 'target_id', 'target_type': 'target_type'}},
+    }
```

Comment:
Consider using inheritance and new clases for this kind of behavior. It's too much responsibility for the same class.